src/app/static/src/app/components/plots/MapControl.vue

```diff
@@ -10,6 +10,7 @@
                         <tree-select :value="indicator"
                                      :multiple="false"
                                      :clearable="false"
+                                     :searchable="false"
```

Comment:
I think it was just a bit confusing to have these searchable (able to be typed) in